package.json

```diff
@@ -18,7 +18,8 @@
       },
       "browser": {
         "types": "./build/lib/index.d.ts",
-        "import": "./build/esm/browser/index.js"
+        "import": "./build/esm/browser/index.js",
+        "default": "./build/esm/browser/index.js"
```

Comment:
`jest@28` updated it's resolver algorithm, jest will now resolve the `"default"` export

Reply:
jest@28 starts using the esm bundle by default? or config will be needed?

Reply:
I'm not sure it prioritizes esm bundle specifically (jest considers it's esm support to be experimental), but it changed the preference of which exports it consumes.
(I turned off the `moduleMapper` config when testing this)

More info here: https://github.com/okta/okta-auth-js/issues/1268